resources/less/style.less

```
@@ -27,3 +25,8 @@
 @import "components/artwork-carousel.less";
 @import "components/zoom-viewer.less";
 @import "components/gallery-map.less";
+
+
+@import "content.less";
+@import "../../node_modules/slick-carousel/slick/slick.css";
```

would something like 

```suggestion
@import "slick-carousel/slick/slick.css";
```

or 

```suggestion
@import "~slick-carousel/slick/slick.css";
```

work